qiskit/circuit/library/__init__.py

```diff
@@ -126,6 +126,42 @@
    ZGate
    GlobalPhaseGate
 
+
+get_standard_gate_name_mapping()
+-------------------
+It returns a dictionary mapping the name of standard gates and instructions to an object for that name.
+
+For Example:
+
+For the identity gate, the dictionary contains a mapping-
+
+.. parsed-literal::
+    'id': Instruction(name='id', num_qubits=1, num_clbits=0, params=[])
+
+Note here that the object on the right belongs to the broad class: 
+
+* :class:`qiskit.circuit.Instruction`
+
+This function returns a map of all the standard gates present in the qiskit library. Instruction object for a particular gate can be captured by indexing the map with the name of the gate.
```

Comment:
linter is failing on this line, because it is too long.
```
qiskit/circuit/library/__init__.py:145:0: C0301: Line too long (188/105) (line-too-long)
```

Reply:
I have separated them into two lines now, so I think it should solve this problem.